src/main/java/core/basesyntax/models/Circle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.models;
```

Comment:
do not use plural in package names
```suggestion
package core.basesyntax.model;
```

Reply:
переименуй пакеты